debezium-server/debezium-server-ybexporter/src/main/java/io/debezium/server/ybexporter/YbExporterConsumer.java

```diff
@@ -79,6 +82,26 @@ void retrieveSourceType(Config config){
         }
     }
 
+    Map<String, String> retrieveColumnSequenceMap(Config config){
+        String propertyVal = PROP_PREFIX + "column_sequence.map";
```

Comment:
sequence_columns.map

Reply:
the map is of <column>:<sequence>, not <sequence>:<column>, therefore I think column_sequence makes more sense here.